client/state/action-types.js

```diff
@@ -693,6 +693,9 @@ export const SITE_WORDPRESS_UPDATE_REQUEST_SUCCESS = 'SITE_WORDPRESS_UPDATE_REQU
 export const SOCIAL_LOGIN_REQUEST = 'SOCIAL_LOGIN_REQUEST';
 export const SOCIAL_LOGIN_REQUEST_SUCCESS = 'SOCIAL_LOGIN_REQUEST_SUCCESS';
 export const SOCIAL_LOGIN_REQUEST_FAILURE = 'SOCIAL_LOGIN_REQUEST_FAILURE';
+export const SOCIAL_CREATE_ACCOUNT_REQUEST = 'SOCIAL_CREATE_ACCOUNT_REQUEST';
```

Comment:
It seems we don’t mention signup anywhere else, the endpoint is called `/users/social/new` so, imo create account is fine, because that’s what we’re doing...
I'll leave it as it is unless you feel strongly about it.